FirebaseTestProject/Views/PostRow.swift

```diff
@@ -40,6 +43,24 @@ struct PostRow: View {
         }
     }
     
+    private var image: some View {
```

Comment:
It might make more sense for this to be a separate structure instead of a property.  Then the semantics are a little nice for adding it:

```
VStack {
    Text("Hi")
    image
    Text("Bye")
}
```

is a little less SwiftUI-looking than

```
VStack {
    Text("Hi")
    PostImage(url)
    Text("Bye")
}
```

Reply:
I agree! I updated the view just now.

Reply:
Great, there's a couple more of these lurking in the codebase, we'll knock them out as we see them.